featured-projects/README.md

```diff
@@ -0,0 +1,17 @@
+# Featured projects
+
+## Lotus
+Lotus is an implementation of the Filecoin Distributed Storage Network specification.
+You can find the Lotus Testground plans [here](https://github.com/filecoin-project/lotus/).
+
+## libp2p
+libp2p is an open source networking stack and library modularized out of The IPFS Project, and bundled separately for other tools to use.
+You can find the libp2p Testground plans [here](https://github.com/libp2p).
+
+## Testground learning example
+This project is intended as a practical example of a "real" project, with its own internal business logic, dependencies, etc. The logic and behaviors are intended to be as straightforward as possible, acting as a reference / guide on how to implement these behaviors and test them using Testground.
```

Comment:
Should there be a link to the project?

Reply:
Nice catch! I must've missed the links when I copied the contents from the gitbook change request.